tasks/main.yml

```diff
@@ -64,6 +64,14 @@
     group="mysql"
     mode=0755
 
+- name: Creates MySQL Error Log File
+  file:
+    path="{{ mysql_log_error }}"
+    state="file"
```

Comment:
https://stackoverflow.com/questions/28347717/how-to-create-an-empty-file-with-ansible

Reply:
updated to "touch" rather than "file"